### DISCUSSION TITLE
GitLab and self signed certificates

### DISCUSSION THREAD

**Discussion by Fatin1234567** [renovatebot/renovate#19255] [Request Help] [answered]
Im using [gitlabRunner](https://gitlab.com/renovate-bot/renovate-runner) to create the renovate bot. I looked at this https://docs.renovatebot.com/examples/self-hosting/#self-signed-tlsssl-certificates but how would I be able detetct private chocolaty package in gitlab. Error 
![image](https://user-images.githubusercontent.com/78020343/205643828-77ce429f-2245-4472-b4d4-3881f0835bf5.png)

**Comment by rarkins** [comment, marked as answer]
Based on that log message it doesn't look like the self signed certificate has been imported correctly. You need to follow the instructions on the page you linked to
